emmaadesile/Ride-My-Way
# #158638006 Setup Travis for continuous integration

### What does this PR do?
Set up Travis-CI for continuous integration

### Description of Task to be completed?
- Setup Travis
- Run all tests in Travis
- Show code coverage report in READ.me

### How should this be manually tested?
After cloning the app, cd into it, run
- `npm install` to install dependencies
- `npm run test` to make sure the tests are passing locally
- push to remote to trigger Travis build

### Any background context you want to provide?
Code climate test coverage report is not showing yet

### What are the relevant pivotal tracker stories?
#158638006
